src/openrct2/drawing/Sprite.cpp

```diff
@@ -648,9 +657,28 @@ extern "C"
             return nullptr;
         }
 
-        if (image_id < SPR_G2_BEGIN)
+        if (image_id < SPR_G1_END)
+        {
+            if (_g1IsRCTC)
+            {
+                return &g1Elements[image_id +
+                                   (image_id >=  1575) * 32 +
```

Comment:
Not really a fan of these bool to int conversions. Makes it pretty hard to read.

Reply:
I understand that. I wasn't particularly a fan before either, but this is both a shorter way of expressing it, and it is well defined behavior (the C++ standard specifies the bool-to-int conversion as false=>0 and true=>1).